src/linagora.esn.calendar/app/settings/import/calendar-import.controller.js

```diff
@@ -58,9 +59,9 @@ function CalCalendarImportController(
 
   function submit() {
     var notificationMessages = {
-      progressing: 'Submitting importing calendar request...',
-      success: 'Request submitted',
-      failure: 'Failed to submit request'
+      progressing: esnI18nService.translate('Submitting importing calendar request'),
```

Comment:
```suggestion
      progressing: esnI18nService.translate('Submitting importing calendar request...'),
```

By the way I'm not sure if you don't need to `esnI18nService.translate('Submitting importing calendar request...').toString()` here.

Reply:
toString() why need to parse as string , what is the difference whith other texts ?

Reply:
@asmaNode the `asyncAction` service translates texts by itself: https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/blob/main/src/frontend/js/modules/async-action.js#L49

Then, we do not need to translate text before calling it.